bundles/org.openhab.persistence.influxdb/src/test/java/org/openhab/persistence/influxdb/internal/ItemToStorePointCreatorTest.java

```diff
@@ -80,6 +80,14 @@ public void convertBasicItem(Number number) {
         return Stream.of(5, 5.5, 5L);
     }
 
+    @Test
```

Comment:
As described in my comment above this test will fail although with the old code, because I think a item name with a dot inside is not valid

https://github.com/openhab/openhab-core/blob/029280683087d07d3d52bc1b125e73c17a33917a/bundles/org.openhab.core/src/main/java/org/openhab/core/common/AbstractUID.java#L96

But in filter criteria Test there is an existing testcase which fails now as well.